core/src/main/java/org/apache/accumulo/core/clientImpl/InstanceOperationsImpl.java

```diff
@@ -232,4 +232,37 @@ public static String lookupInstanceName(ZooCache zooCache, UUID instanceId) {
     }
     return null;
   }
+
+  /**
+   * Returns a unique string that identifies this instance of accumulo.
+   *
+   * @return a String
+   */
+  @Override
+  public String getInstanceID() {
+    context.ensureOpen();
```

Comment:
It seems like it may be better to leave the implementation of getInstanceID() in Client context.  Then the private members of client context would not need to be exposed.  This method could then `return context.getInstanceID()`.